bases/shared/entrypoint.sh

```diff
@@ -870,6 +870,7 @@ case "$ROLE" in
             ( wait_till_syncup_and_fund ) &
         fi
 
+        /bin/bash /entrypoint/cron.sh
```

Comment:
Cron runs jobs in the background by default and hence we haven't explicitly invoked `/entrypoint/cron.sh` with `&`, Right?

Reply:
Yes